app-android-xamarin/Moodis/Moodis/Resources/layout/activity_menu.xml

```diff
@@ -0,0 +1,173 @@
+﻿
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ImageView
+        android:id="@+id/imageForView"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/grid_unit_190x"
+        android:scaleType="centerCrop"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+    </ImageView>
+
+    <TextView
+        android:id="@+id/lblAnger"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/grid_unit_4x"
+        android:layout_marginLeft="@dimen/grid_unit_4x"
+        android:layout_marginTop="@dimen/grid_unit_2x"
+        android:layout_marginEnd="@dimen/grid_unit_56x"
+        android:layout_marginRight="@dimen/grid_unit_56x"
+        android:text="@string/placeholder"
```

Comment:
there is `tools:text` attribute, which is specially designed to be used when you need to have placeholder text during development. It will be deleted when the app is built.